shacl/validate.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
```

Comment:
This should probably be a Makefile rather than a shell script. Follow the example of the Makefile in the root of this repository to avoid regenerating Brick or SHACL shapes when the source files haven't changed

Reply:
I thought about a makefile but I generally don't like make's way of passing arguments arg=xxx.  It's clumsy for the user (not auto-completion).  Since generating Brick.ttl and shacl_test.ttl is pretty fast (pyshacl is the slow one), I'd say we leave it like this until we find the speed intolerable.